include/swift/SIL/DynamicCasts.h

```diff
@@ -448,6 +448,13 @@ struct SILDynamicCastInst {
     return t->getCanonicalType();
   }
 
+  Optional<SILType> getLoweredBridgedTargetObjectType() const {
+    CanType t = getBridgedTargetType();
+    if (!t)
+      return None;
+    return SILType::getPrimitiveObjectType(t);
```

Comment:
This can be reduced to:
```
  if (CanType T = getBridgedTargetType())
    return SILType::getPrimitiveObjectType(T);
  return None;
```

Reply:
This is just a matter of taste. I prefer None to be an early exit.